modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/selected-quote/services/DocumentsAndMedia.js

```diff
@@ -28,6 +28,10 @@ export const createDocumentInFolder = async (folderId, fileEntry, callback) => {
 	);
 };
 
+export const RemoveDocument = async (documentId) => {
```

Comment:
Replace RemoveDocument to removeDocumentById or deleteDocumentById

Reply:
You can also remove this async keyword, since you are not resolving this promise